examples/tcpblaster/tcpblaster_client.c

```diff
@@ -166,7 +166,6 @@ void tcpblaster_client(void)
 
       /* Wait until we can send data or until the connection is lost */
 
-      printf("=");
```

Comment:
Modify the previous patch instead, it's better to make the patch clean if it doesn't merge into mainline yet.

Reply:
@xiaoxiang781216 ok

Reply:
It's very easy, all is automatically with git rebase --interactive:
https://hackernoon.com/beginners-guide-to-interactive-rebasing-346a3f9c3a6d
Once your finish to refine the patchset, you can run git push with -f flag to update the PR.

Reply:
@xiaoxiang781216 rebase against master and push -f, got it. Thanks. I will fix this tomorrow!